pkgs/development/misc/avr-gcc-with-avr-libc/default.nix

```diff
@@ -26,8 +26,8 @@ stdenv.mkDerivation {
   
   buildInputs = [ gmp mpfr libmpc zlib ];
   
-  # Make sure we don't strip the libraries in lib/gcc/avr.
-  stripDebugList= [ "bin" "avr/bin" "libexec" ];
+  # Make sure we don't strip the libraries in lib/gcc/avr and in avr/lib.
```

Comment:
Because if stripped linking with them tends to fail, presumably because there are some functionally relevant symbols there. Can't say more.